### ISSUE TITLE
Bug: Older versions of onedrive pulled from Docker Hub

### ISSUE THREAD

**Issue by OscarVanL** [abraunegg/onedrive#2679]
### Describe the bug

It looks like the Docker Hub image I am reliant on has been pulled:

`ERROR: failed to solve: driveone/onedrive:2.4.24-debian: docker.io/driveone/onedrive:2.4.24-debian: not found`

Please can you reinstate this? Currently I am blocked on updating to the latest version.

### Operating System Details

```shell
Debian-based container
```

### How do you use 'onedrive'

I run onedrive containerised on a server to sync files to sharepoint.

### Steps to reproduce the behaviour

Try to pull `driveone/onedrive:2.4.24-debian`

### Complete Verbose Log Output

```shell
Building onedrive container (DEBUG: 0)
#0 building with "default" instance using docker driver

#1 [internal] load build definition from onedrive.Dockerfile
#1 transferring dockerfile: 739B done
#1 DONE 0.0s

#2 [internal] load metadata for docker.io/driveone/onedrive:2.4.24-debian
#2 ...

#3 [auth] driveone/onedrive:pull token for registry-1.docker.io
#3 DONE 0.0s

#2 [internal] load metadata for docker.io/driveone/onedrive:2.4.24-debian
#2 ERROR: docker.io/driveone/onedrive:2.4.24-debian: not found
------
 > [internal] load metadata for docker.io/driveone/onedrive:2.4.24-debian:
------
onedrive.Dockerfile:3
--------------------
   1 |     # https://github.com/abraunegg/onedrive
   2 |     # https://hub.docker.com/r/driveone/onedrive
   3 | >>> FROM driveone/onedrive:2.4.24-debian
   4 |     
   5 |     # Install jq dependency required for structured logging
--------------------
ERROR: failed to solve: driveone/onedrive:2.4.24-debian: docker.io/driveone/onedrive:2.4.24-debian: not found
```

**Comment by abraunegg** [comment]
@OscarVanL 
Unfortunatly that is not going to be possible - sorry.

You will have to update your build process to use v2.4.25 or use the v2.4.24 release code and build your own v2.4.24 Debian Docker image.

Please also note that v2.5.0 is probably 4 to 6 weeks away from release, thus v2.4.25 images will also be purged after a short period of time.

**Comment by abraunegg** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
